### PR TITLE
Fix tests on Windows

### PR DESCRIPTION
**Pull request by MiniGod**
Was unable to run `npm test` on windows.

### CODE REVIEW

**Comment by kristjanmik** [comment]
Since no collaborator has the ability to run this other than you @MiniGod I will trust your work on this.

File: package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "apis-helpers": "0.0.1"
   },
   "scripts": {
-    "test": "node_modules/mocha/bin/mocha test/integration",
+    "test": "mocha test/integration",
```
Comment (koddsson) [inline package.json]: This means that you have to install `mocha` globally. Can we add that somehow in the `package.json` or do we just need to include that in the `README` setup instructions?

Comment (MiniGod) [inline package.json]: You need to install globally. `npm` is smart enough to look through your `node_modules` for binaries used in the `scripts` section in `package.json`.

Comment (kristjanmik) [inline package.json]: What about running ./test and do some checking if mocha is globally available, else default to the old one with correct seperator for windows and linux?

Comment (koddsson) [inline package.json]: >You don't need to install globally. npm is smart enough to look through your node_modules for binaries used in the scripts section in package.json.

Oh cool.

**Pull request merged**